modules/ossm-config-otel-kiali.adoc

```diff
@@ -23,22 +23,18 @@ After {KialiProduct} is integrated with {DTProductName}, you can view distribute
 ----
 spec:
   external_services:
-    grafana:
-      enabled: true # <1>
-      in_cluster_url: http://grafana-istio-system.apps-crc.testing/ # <2>
-      url: http://grafana-istio-system.apps-crc.testing/
     tracing:
-      enabled: true
+      enabled: true #<1>
       provider: tempo
       use_grpc: false
       in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
-      url: https://tempo-sample-query-frontend-tempo.apps-crc.testing
-      tempo_config: # <3>
+      url: https://tempo-sample-query-frontend-tempo.apps-crc.testing #<2>
```

Comment:
This won't be needed

Reply:
Isn't that URL ( in `.tracing.url`) used in `Distributed Tracing` link in Kiali UI ? ( and when it is not set, the `Distributed Tracing` link is missing in Kiali UI ? ) 
![image](https://github.com/user-attachments/assets/bc13d475-8d47-4c43-9cf8-b6f316121dc8)

( or does it work that way only for KIali 1.73? and for 1.89> it works differently? [I have not tried ossm3 + kiali 1.89 + tempo yet, I am going to do today] )

Reply:
Ah right @mkralik3 , I've missed this. Let me update the comments